tox.ini

```diff
@@ -1,32 +1,17 @@
 [tox]
+isolated_build = true
 envlist =
-    py36-django{31,32,tip},
-    py37-django{31,32,tip},
-    py38-django{31,32,tip}
-    py39-django{31,32,tip}
-    migrationscheck
+    py{35,36,37,38,39}-django22
```

Comment:
I forgot to mention, I did add back in testing for Django 2.2 as it's LTS and should be something the project supports through 1H 2022.